openpaas-james/apps/distributed-es6-backport/src/main/conf/elasticsearch.properties

```diff
@@ -0,0 +1,83 @@
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+#  This template file can be used as example for James Server configuration
+#  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
+
+# Configuration file for ElasticSearch
+# Read https://james.apache.org/server/config-elasticsearch.html for further details
+
+elasticsearch.masterHost=elasticsearch
+elasticsearch.port=9200
+
+# Optional. Only http or https are accepted, default is http
+# elasticsearch.hostScheme=http
+
+# Optional, default is `default`
+# Choosing the SSL check strategy when using https scheme
+# default: Use the default SSL TrustStore of the system.
+# ignore: Ignore SSL Validation check (not recommended).
+# override: Override the SSL Context to use a custom TrustStore containing ES server's certificate.
+# elasticsearch.hostScheme.https.sslValidationStrategy=default
+
+# Optional. Required when using 'https' scheme and 'override' sslValidationStrategy
+# Configure Elasticsearch rest client to use this trustStore file to recognize nginx's ssl certificate.
+# You need to specify both trustStorePath and trustStorePassword
+# elasticsearch.hostScheme.https.trustStorePath=/file/to/trust/keystore.jks
+
+# Optional. Required when using 'https' scheme and 'override' sslValidationStrategy
+# Configure Elasticsearch rest client to use this trustStore file with the specified password.
+# You need to specify both trustStorePath and trustStorePassword
+# elasticsearch.hostScheme.https.trustStorePassword=myJKSPassword
+
+# Optional. default is `default`
+# Configure Elasticsearch rest client to use host name verifier during SSL handshake
+# default: using the default hostname verifier provided by apache http client.
+# accept_any_hostname: accept any host (not recommended).
+# elasticsearch.hostScheme.https.hostNameVerifier=default
+
+# Optional.
+# Basic auth username to access elasticsearch.
+# Ignore elasticsearch.user and elasticsearch.password to not be using authentication (default behaviour).
+# Otherwise, you need to specify both properties.
+# elasticsearch.user=elasticsearch
+
+# Optional.
+# Basic auth password to access elasticsearch.
+# Ignore elasticsearch.user and elasticsearch.password to not be using authentication (default behaviour).
+# Otherwise, you need to specify both properties.
+# elasticsearch.password=secret
+
+# You can alternatively provide a list of hosts following this format :
+# elasticsearch.hosts=host1:9200,host2:9200
+# elasticsearch.clusterName=cluster
+
+elasticsearch.nb.shards=5
+elasticsearch.nb.replica=1
+elasticsearch.index.waitForActiveShards=1
+elasticsearch.retryConnection.maxRetries=7
+elasticsearch.retryConnection.minDelay=3000
+# Index or not attachments (default value: true)
+elasticsearch.indexAttachments=true
+
+# Reports for metrics into ElasticSearch
```

Comment:
Unrelated but we can drop the metric part of this file in TMail as we do not rely on the ElasticSearch metric reporter...